OrbitGl/GlCanvas.cpp

```diff
@@ -27,15 +27,20 @@ float GlCanvas::kZValueOverlayTextBackground = 0.45f;
 float GlCanvas::kZValueText = 0.47f;
 float GlCanvas::kZValueEventBarPicking = 0.49f;
 float GlCanvas::kZValueUi = 0.61f;
-float GlCanvas::kZValueTimeBarBg = 0.63f;
-float GlCanvas::kZValueTextUi = 0.65f;
-float GlCanvas::kZValueMargin = 0.81f;
-float GlCanvas::kZValueSliderBg = 0.83f;
-float GlCanvas::kZValueSlider = 0.85f;
+float GlCanvas::kZValueTextUi = 0.61f;
+float GlCanvas::kScreenSpaceCutPoint = 0.8f;
+float GlCanvas::kZValueTimeBarBg = 0.81f;
+float GlCanvas::kZValueTimeBar = 0.83f;
+float GlCanvas::kZValueMargin = 0.85f;
+float GlCanvas::kZValueSliderBg = 0.87f;
+float GlCanvas::kZValueSlider = 0.89f;
 
 float GlCanvas::kZOffsetMovingTack = 0.1f;
 float GlCanvas::kZOffsetPinnedTrack = 0.2f;
 
+// Max Number of layers: 16 original, 4 for moving track, 4 for pinned Track
+unsigned GlCanvas::kMaxNumberRealZLayers = 16 + 4 + 4;
```

Comment:
This is going to be a pain to maintain, ideally we could compute the value dynamically instead of hard-coding it. Let's get your PR in, but we should think about how we could improve this.

Reply:
Yes, it's not very nice. But we don't know if we are going to still have this constant in the future, or make a different approach like using enum. For now, I couldn't think something better.